packages/core/core/src/Strapi.ts

```diff
@@ -166,23 +142,20 @@ class Strapi extends Container implements Core.Strapi {
 
     // Instantiate the Strapi container
     this.add('config', registries.config(appConfig, this))
-      .add('content-types', registries.contentTypes())
-      .add('components', registries.components())
-      .add('services', registries.services(this))
-      .add('policies', registries.policies())
-      .add('middlewares', registries.middlewares())
-      .add('hooks', registries.hooks())
-      .add('controllers', registries.controllers(this))
-      .add('modules', registries.modules(this))
-      .add('plugins', registries.plugins(this))
-      .add('custom-fields', registries.customFields(this))
-      .add('apis', registries.apis(this))
-      .add('sanitizers', registries.sanitizers())
-      .add('validators', registries.validators())
       .add('query-params', createQueryParamService(this))
       .add('content-api', createContentAPI(this))
       .add('auth', createAuth())
-      .add('models', registries.models());
+      .add('server', () => createServer(this))
```

Comment:
would it make sense to have a type registry for the strapi containers? 
As far as I know doing strapi.get('containerName') does not return a typed value(?)

Reply:
Yes we will need to figure out a nice way to type it. Usually with di you type at loading but I don't like it, with the shortcut we maintain the all access with the types for now